### DISCUSSION TITLE
Crash on Resetting game objects when creating or starting Medium or Large worlds (pi 3)

### DISCUSSION THREAD

**Discussion by Alexc99xd** [Pryaxis/TShock#2931] [General]
Hi, I am running TShock on a Pi 3, aarch64 (arm 64)

When creating or start worlds that are medium or larger, TShock gets killed. I've tried importing medium worlds to start those worlds but it didn't work. Small worlds create and run fine.
I tried adding -disable-commands flag but no avail

Could this just be a ram and processor issue?

only things in the logs are INFO loggers

**Comment by sgkoishi** [comment]
You can run something like `top` to monitor. Likely yes for out of RAM, try [Constileation](https://ikebukuro.tshock.co/#/tile-providers).

**Reply by Alexc99xd** [reply]
`-c`

 flag made it work! Thanks!
